### PR TITLE
Ruby 3 upgrade

### PR DESCRIPTION
**Pull request by Sangeetha-Bheeman**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -45,6 +45,9 @@ Naming/MemoizedInstanceVariableName:
 Style/RedundantInitialize:
   Enabled: false
 
+Style/HashSyntax:
+  Enabled: false
```
Comment (sdjmchattie) [inline .rubocop.yml:49]: Any reason why we don't want this?  You can choose our preferred style by setting the `EnforcedStyle` ... https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax

I would think `no_mixed_keys` gives a pretty good result as we shouldn't be mixing key types.

**Review by sdjmchattie — APPROVED** [review]
One question about Rubocop configuration.  We should avoid turning off rules unless there's a good reason to.

File: .rubocop.yml (hunk 1 above)
Comment (Sangeetha-Bheeman) [reply .rubocop.yml:49]: Okay, I will change that.

**Comment by emrojo** [comment]
Tests done in uat: 

https://docs.google.com/spreadsheets/d/1lu7szTfkJctYpTqweikcp6EMQu69nlLixvLdlHhwcGk/edit#gid=0

All good

**Pull request merged**